rclcpp/include/rclcpp/subscription_base.hpp

```diff
@@ -491,6 +491,38 @@ class SubscriptionBase : public std::enable_shared_from_this<SubscriptionBase>
     event_handlers_[event_type]->clear_on_ready_callback();
   }
 
+#ifdef CONTENT_FILTER_ENABLE
+  /// Set the filter expression and expression parameters for the subscription.
```

Comment:
We should explain how to clear the content filtering as input arguments or we could have clear method for user application?

Reply:
confirmed this is addressed in header doc below.